app/cmd/replica.go

```diff
@@ -44,11 +44,11 @@ func ReplicaCmd() cli.Command {
 			},
 			cli.StringFlag{
 				Name:  "restore-from",
-				Usage: "specify backup to be restored, must be used with --restore-name",
+				Usage: "specify backup to be restored, must use with --restore-name",
```

Comment:
since there is no subject in the sentence here, it's better to keep `A must be used with B`.